docs/tutorials/kev-dev-with-skaffold.md

```diff
@@ -0,0 +1,97 @@
+---
+weight: 11
+title: Kev dev with skaffold
+---
+
+# Kev Dev w/Skaffold
+
+This tutorial will walk you through how to iterate on Kubernetes application using _Kev_ `dev` command with an optional [Skaffold](https://skaffold.dev/) hook.
+
+We encourage everyone to get familiar with [Getting started with Kev](getting-started-with-kev.md) tutorial before proceeding with this guide.
+
+## Iterate Docker Compose
+
+As you've learnt in the [Getting Started](getting-started-with-kev.md) guide, you can turn any Docker Compose project into a set of Kubernetes manifests
+with simple `kev render` command.
+
+However, when frequent changes are made to any of the source or environment specific compose files, this may be a little cumbersome.
+
+To automate the process of rendering K8s manifests, run _Kev_ in development mode (see command [reference](cli/kev_dev.md) for details)
+
+> Run Kev in dev mode. Starts watch loop an automatically re-renders K8s manifests.
+```sh
+# for all environments
+kev dev
+
+# for specified environment(s)
+kev dev -e myenv
+````
+
+It will start watch loop over source compose & environment override files. When modification is detected it'll automatically re-render Kubernetes manifests for environments specified via `--environment | -e` flag(s). If no environments have been specified it'll automatically default to `dev` environment.
+
+## Automatic Develop / Build / Push / Deploy
+
+This section will describe how to take advantage of existing Development Lifecycle tools enhancing developer experience when iterating on the Kubernetes application locally. We'll focus on [Skaffold](https://skaffold.dev/) and _Kev_ integration and the experience they both offer together.
+
+## ⚬ Initialise Kev project with Skaffold support
+
+In order to take advantage of Skaffold, you must prepare your project accordingly. Provided you haven't initialised Kev application yet it's as simple as running the following command:
+
+> Run Kev in dev mode with Skaffold dev loop enabled.
```

Comment:
(copy/paste mistake?) Feels like the wrong place for this info as you're about to discuss how to `init`...